src/components/SideBar/FollowButton.tsx

```diff
@@ -1,29 +1,32 @@
 import { Button } from "@mui/material";
-import React from "react";
 
 const styles = {
   followButton: {
     boxShadow: "none",
     backgroundColor: "primary.main",
     "&:hover": {
       boxShadow: "none",
-      backgroundColor: "#1E9964", // could be primary.dark
+      backgroundColor: "#1E9964", // Could be primary.dark
     },
   },
 };
 
-const FollowButton = () => {
-  const handleFollowButton = (e: React.MouseEvent) => {
+type FollowButtonProps = {
+  onClick?: () => void;
```

Comment:
I made this prop as an optional one in case you wanted to add an onClick handler to the button. Sometimes you also want other actions done when someone clicks the button so this can be used for that